cmd/frontrunning/main.go

```diff
@@ -20,50 +19,77 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/ethclient/gethclient"
 	"github.com/ethereum/go-ethereum/rlp"
-	"github.com/ethereum/go-ethereum/rpc"
 )
 
 func main() {
-	ethclient, err := ethclient.Dial("https://goerli.infura.io/v3/" + utils.GetKey("INFURA_KEY"))
+	ethclient, err := ethclient.Dial("https://sepolia.infura.io/v3/" + utils.GetKey("INFURA_KEY"))
 	if err != nil {
-		log.Fatal("[0] ", err)
-
+		log.Fatal("Client not connected.", err)
 	}
 
-	rpc, err := rpc.Dial("wss://goerli.infura.io/ws/v3/" + utils.GetKey("INFURA_KEY"))
-	if err != nil {
-		log.Fatal("[1] ", err)
-	}
-	subscriber := gethclient.New(rpc)
+	// rpc, err := rpc.Dial("wss://sepolia.infura.io/ws/v3/" + utils.GetKey("INFURA_KEY"))
+	// if err != nil {
+	// 	log.Fatal("Client not connected. ", err)
+	// }
+	// subscriber := gethclient.New(rpc)
 	fmt.Println("client connected..")
 
 	// txs := make(chan *types.Transaction)
 	// _, err = rpcClient.SubscribeFullPendingTransactions(context.Background(), txs)
 	// if err != nil {
 	// 	panic(err)
 	// }
+	from := common.HexToAddress("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266")
+	nonce, _ := ethclient.PendingNonceAt(context.Background(), from)
+	suggestedGasPrice, _ := ethclient.SuggestGasPrice(context.Background())
+	gasPrice := new(big.Int).Mul(suggestedGasPrice, big.NewInt(10))
+	gas := uint64(30000000)
+	to := common.HexToAddress("0x976EA74026E726554dB657fA54763abd0C3a0aa9")
```

Comment:
what is this address?

Reply:
It's one of the accounts addresses anvil provided